stackdriver_exporter.go

```diff
@@ -236,7 +240,14 @@ func (h *handler) innerHandler(filters map[string]bool) http.Handler {
 	}
 
 	// Delegate http serving to Prometheus client library, which will call collector.Collect.
-	return promhttp.HandlerFor(gatherers, promhttp.HandlerOpts{})
+	opts := promhttp.HandlerOpts{}
```

Comment:
This can be simplified to:
```suggestion
	opts := promhttp.HandlerOpts{ErrorLog: stdlog.New(log.NewStdlibAdapter(level.Error(h.logger)), "", 0)}
```

There's no need to have a new flag for this, just adding the `ErrorLog` handler to promhttp is enough.